src/app/static/src/app/store/root/mutations.ts

```diff
@@ -158,3 +164,11 @@ export const mutations: MutationTree<RootState> = {
     ...languageMutations
 
 };
+
+const stopPolling = (state: any) => {
```

Comment:
Is this the idiomatic way to do this? I want to pass in state which could be `ModelCalibrateState` or `ModelRunState` or one of the download states

Reply:
I think it would be more typical in Typescript to make an interface which all these state types support and define that as being the type passed in here. 
```
interface PollingState {
    statusPollId: number
}
```
... and then update each of the relevant state types such that they extend the interface ...and remove `statusPollId` as defined indivudally in each of those,,

I'm surprised that use of `any` isn't failing linting, I thought we'd disallowed it in non-test code.

Reply:
I think the type interface `PollingState` is the right approach but instead of changing the individual states of the download results I think it's simpler to use generics and do this for the type definition of `stopPolling`
```TypeScript
const stopPolling = <T extends PollingState>(state: T) => {
```
This will make sure anything passed into it has the `statusPollId` field!